versioning/README.md

```diff
@@ -1,14 +1,14 @@
-## versioning
+## Versioning
 
-This dir contains a common versioning script which prints the version of the current git commit to stdout.
+This package contains a script that prints the semantic version of the current git commit to stdout.
 
 This script is a wrapper around the functionality of `git describe --tags --dirty` with small changes to split the git pre-release identifiers to allow
 semver sorting.
-It requires the latest tag of the current branch to be a semver version, otherwise it raises an exception.
-Semver versions with plus elements like `1.0.2+gold` are not supported and also raise an exception.
+It requires the latest tag of the current branch to be a semantic version, otherwise it raises an exception.
+Semantic versions with plus elements like `1.0.2+gold` are not supported and also raise an exception.
 
-If the latest commit is not tagged by a semver tag then a pre-release version is printed.
+If the latest commit does not point to a semver tag, then a pre-release version is printed.
```

Comment:
I disagree with this change: a commit does not point to a tag, a tag points to a commit.

A commit can exist without a tag, but a tag always needs a commit to point to.

Also, there can be many tags pointing to the same commit, but each tag can only point to a single commit.

So the concept of a commit pointing to a tag doesn't really work for me.

Side note: now I'm wondering what results we should expect from the versioning package if multiple semantic version tags point at the current commit?